service-netty/pom.xml

```diff
@@ -28,5 +28,10 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>1.3.2</version>
+        </dependency>
```

Comment:
Is it really required to add this dependency. I'm not against adding new dependencies, but once it is added it is very difficult to remove in the future.